src/pages/audio-rewards-page/store/sagas.ts

```diff
@@ -179,6 +180,10 @@ function* claimChallengeRewardAsync(
               setVisibility({ modal: COGNITO_MODAL_NAME, visible: true })
             )
             break
+          case FailureReason.ALREADY_DISBURSED:
```

Comment:
We should also handle the `ALREADY_SENT` error - this one comes from the reward manager, and you'll see if it you submit after the challenge has completed but before DN has indexed it.